### PR TITLE
[BACKLOG-33360] - Implemented VFS browser for Excel Writer Step for b…

### PR DESCRIPTION
**Pull request by wseyler**
…oth the output and template location selections

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 21m 10s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl ui
```
---
:ok_hand: All tests passed!
 
Tests run: **1139**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7298/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: ui/src/main/java/org/pentaho/di/ui/trans/steps/excelwriter/ExcelWriterStepDialog.java [hunk 1]
```diff
@@ -1471,43 +1475,33 @@ public void modifyText( ModifyEvent e ) {
...
-        }
-      }
-    } );
+    wbFilename.addSelectionListener(
+      new SelectionAdapterFileDialogTextVar(
+        log,
+        wFilename,
+        transMeta,
+        new SelectionAdapterOptions(
+          SelectionOperation.SAVE,
```
[the remark this answers is not in the document]
Comment (wseyler) [reply ui/src/main/java/org/pentaho/di/ui/trans/steps/excelwriter/ExcelWriterStepDialog.java]: Changes made.

File: ui/src/main/java/org/pentaho/di/ui/trans/steps/excelwriter/ExcelWriterStepDialog.java [hunk 2]
```diff
@@ -1471,43 +1475,33 @@ public void modifyText( ModifyEvent e ) {
...
+      )
+    );
+
+    wbTemplateFilename.addSelectionListener(
+      new SelectionAdapterFileDialogTextVar(
+        log,
+        wTemplateFilename,
+        transMeta,
+        new SelectionAdapterOptions(
+          SelectionOperation.OPEN,
```
[the remark this answers is not in the document]
Comment (wseyler) [reply ui/src/main/java/org/pentaho/di/ui/trans/steps/excelwriter/ExcelWriterStepDialog.java]: Changes made.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 22m 28s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl ui
```
---
:ok_hand: All tests passed!
 
Tests run: **1139**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7298/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**